### PR TITLE
add from_stl classmethod to the Mesh object

### PR DESCRIPTION
**Pull request by devngc**

### CODE REVIEW

**Comment by devngc** [comment]
cc: @chriswmackey

File: ladybug_geometry/geometry3d/_stl.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+    def _load_stl(self):
+        """Load the ASCII STL file and extract triangular faces."""
+
+        assert os.path.isfile(self._file_path), 'Failed to find %s' % self._file_path
+
+        with open(self._file_path, 'rb') as fp:
+            h = fp.read(80)
+            first_word = h[0:5]
+            fp.close()
```
Comment (mostaphaRoudsari) [inline ladybug_geometry/geometry3d/_stl.py:43]: When you use `with` clause you don't need to close the file anymore. It does it for you.

File: ladybug_geometry/geometry3d/mesh.py [hunk 2]
```diff
@@ -1,10 +1,12 @@
 # coding=utf-8
 """3D Mesh"""
+
 from __future__ import division
 
+import struct
```
Comment (mostaphaRoudsari) [inline ladybug_geometry/geometry3d/mesh.py:6]: Do you need to import this here?

**Review by mostaphaRoudsari — COMMENTED** [review]
See my comments.

**Comment by chriswmackey** [comment]
Thank you, @devngc . I ended up working from what you had here as it was decent. There were just a few implementation differences that I wanted to make and I also noticed that the binary STL importer always results in the last face being duplicated. This appears to be a bug in the original implementation.

In any event, this has all been fixed in [this new PR](https://github.com/ladybug-tools/ladybug-geometry/pull/340) so I am closing this one.

**Pull request closed without merging**